vault/README.adoc

```diff
@@ -1,34 +1,36 @@
-# Plugin for Vault and Rudder
+Plugin for Vault and Rudder
 
-This project is part of Rudder - IT infrastructure Automation and Compliance.
-See: http://rudder.io for more information.
+This project is part of Rudder - IT infrastructure Automation and Compliance. See: http://rudder.io for more information.
 
-See repository main README for information about licensing, contribution, and
-other general information.
+See repository main README for information about licensing, contribution, and other general information.
 
-// Everything after this line goes into Rudder documentation
-// ====doc====
-[vault-plugin]
-= Vault
```

Comment:
The `=` sign in asciidoc is a first level title. Why removing it?

Reply:
looks like i removed it by mistake! done re adding it!